package.json

```diff
@@ -23,6 +23,7 @@
     "scripts/*"
   ],
   "scripts": {
+    "test": "lerna run test --ignore demo-*",
```

Comment:
😎

Reply:
non-blocking: Per #75 may want to consider switching to `scope` instead of `ignore`